src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/UpdateValidatorTest.java

```diff
@@ -70,15 +98,79 @@ void testValidationErrorReportedWhenFilingDateIsInFuture() {
     }
 
     @Test
-    void testNoFullValidationErrorReportedWhenFilingDateIsInPast() {
+    void testNoValidationErrorReportedWhenFilingDateIsBeforeNextMadeUpToDate() throws ServiceException {
```

Comment:
is this test effectively the same as testNoValidationErrorReportedWhenFilingDateIsNotNull? wondering if testNoValidationErrorReportedWhenFilingDateIsNotNull is needed.

Reply:
Yes, I think the `testNoValidationErrorReportedWhenFilingDateIsNotNull` test can go as plenty of (new) tests now covering this scenario.